conduct/enforcement.md

```diff
@@ -83,6 +84,28 @@ satisfactory resolution was achieved or not) and decide on any additional
 action deemed necessary.
 
 
+## Acting unilaterally
+
+In some cases, individuals may violate the CoC in online or in-person situations.
+If any community member deems the action to be harmful to the community,
+they may act immediately, before reaching consensus, to defuse the situation.
+In ongoing situations, any member may at their discretion employ any of the
+tools available in this enforcement manual, including bans and blocks online,
+or removal from a physical space.
+
+If the incident involves physical danger, or involves a threat to anyone's safety
+(e.g. threats of violence), any member of the community may -- and should -- act
+unilaterally to protect the safety of any community member.
+This can include contacting law enforcement (or other local personnel) and
+speaking on behalf of the Jupyter Project.
```

Comment:
I think this language is a bit too broad. Can we delete the "speaking on behalf of the Jupyter Project"? I think it makes sense to reserve that designation to communications that really are from the project steering council, rather than unilateral actions by a member of the community.

Reply:
good point - I think that in the future and depending on how the governance conversation hashes out, the "speak on behalf of Jupyter" point could be specifically given to "core team members" as opposed to generically saying "community member" but for now I'll remove that language